executioncontext/projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/model/sections/ProjectViewSingletonSection.kt

```diff
@@ -20,6 +20,13 @@ data class ProjectViewDebuggerAddressSection(override val value: String) :
     }
 }
 
+data class ProjectViewDeriveTargetsFlagSection(override val value: Boolean) :
```

Comment:
lets keep the convention with 1-1 mapping 
```suggestion
data class ProjectViewDeriveTargetsFromDirectoriesSection(override val value: Boolean) :
```

Reply:
Fixed